### PR TITLE
neogeo.xml: Added 1 working item

### PR DESCRIPTION
**Pull request by ArcadeShadow**
New working software list addition
--------------------------------------------
GladMort (demo 2) [PixelHeart]

### CODE REVIEW

File: hash/neogeo.xml [hunk 1]
```diff
@@ -11622,7 +11622,36 @@ license:CC0-1.0
 	</software>
 
 	<software name="gladmort">
-		<description>Gladmort (demo)</description>
+		<description>GladMort (demo 2)</description>
```
Comment (angelosa) [inline hash/neogeo.xml:11625]: What `demo 2` is supposed to mean, versioning?

Comment (ArcadeShadow) [reply hash/neogeo.xml:11625]: It's a new available demo, which title contains "demo 2"

Comment (angelosa) [reply hash/neogeo.xml:11625]: It's supposed to be `(demo²)` then.

Comment (ArcadeShadow) [reply hash/neogeo.xml]: > What `demo 2` is supposed to mean, versioning?

OK. Fair enough.
This means that the Amiga CD32 should also be written with superscript style?
image

Comment (happppp) [reply hash/neogeo.xml]: No. Unless maybe the MAME Amiga driver author(s) insists on it. Likewise, we don't have Odyssey².

Comment (angelosa) [reply hash/neogeo.xml]: It's fine for me, yes, assuming it's changed in `amiga/amiga.cpp` too.

**Pull request merged**